pkg/image/apis/image/helper.go

```diff
@@ -494,9 +494,9 @@ func ImageWithMetadata(image *Image) error {
 			image.DockerImageLayers[i].MediaType = layer.MediaType
 		}
 		// reverse order of the layers for v1 (lowest = 0, highest = i)
-		for i, j := 0, len(image.DockerImageLayers)-1; i < j; i, j = i+1, j-1 {
-			image.DockerImageLayers[i], image.DockerImageLayers[j] = image.DockerImageLayers[j], image.DockerImageLayers[i]
-		}
+		// for i, j := 0, len(image.DockerImageLayers)-1; i < j; i, j = i+1, j-1 {
```

Comment:
@smarterclayton I think this schema2 reordering is wrong. schema1 and schema2 should have same order in our database.